py_experimenter/experimenter.py

```diff
@@ -282,68 +296,110 @@ def execute(self, experiment_function: Callable[[dict, dict, ResultProcessor], N
         :type random_order: bool, optional
         :raises InvalidValuesInConfiguration: If any value of the experiment parameters is of wrong data type.
         """
-        logging.info("Start execution of experiment_functions...")
-        keyfield_values = self.dbconnector.get_keyfield_values_to_execute()
-
-        if random_order:
-            shuffle(keyfield_values)
-
-        if 0 <= max_experiments < len(keyfield_values):
-            keyfield_values = keyfield_values[:max_experiments]
-        result_field_names = utils.get_result_field_names(self.config)
         try:
-            cpus = int(self.config['PY_EXPERIMENTER']['cpu.max'])
+            n_jobs = int(self.config['PY_EXPERIMENTER']['n_jobs'])
         except ValueError:
-            raise InvalidValuesInConfiguration('cpu.max must be an integer')
-        table_name = self.get_config_value('PY_EXPERIMENTER', 'table')
-        result_processors = [ResultProcessor(self.config, self.database_credential_file_path, table_name=table_name, condition=p,
-                                             result_fields=result_field_names) for p in keyfield_values]
-        experiment_functions = [experiment_function for _ in keyfield_values]
-        try:
-            custom_fields = [dict(self.config.items('CUSTOM')) for _ in keyfield_values]
-        except NoSectionError:
-            custom_fields = [None for _ in keyfield_values]
+            raise InvalidValuesInConfiguration('n_jobs must be an integer')
```

Comment:
`n_jobs` is an optional argument, in case it is not given in the config, it is set to `1` per default